spring-cloud-gcp-examples/spring-cloud-gcp-config-example/README.adoc

```diff
@@ -4,15 +4,17 @@ This sample application demonstrates using the
 link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-config[Spring Cloud GCP Config Client Starter] in your code.
 
 == Setup & Configuration
-1. Follow link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-config[these setup instructions].
-2.  Create a configuration using the
+1. Checkout the code of this sample.
+2. Setup the Google Runtime Configuration API with
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-config#setup[these instructions].
```

Comment:
Why don't use *Internal links* support in ASCII Doc?

Reply:
If you mean the `<<>>` links, I don't think they can be used because the links would be in another document. Let me know if I'm missing anything.

Reply:
??? We have one big HTML or PDF in the end. What is the "another document" are you referring to?